src/hdmf/term_set.py

```diff
@@ -12,19 +15,37 @@ class TermSet():
     :ivar view: SchemaView of the term set schema
```

Comment:
Please add `:ivar ...:` docs to the class docstring for new instance variables ``self.schemasheets_folder`` and ``self.expanded_term_set_path``